dashboard/app/types/types.ts

```diff
@@ -1,14 +1,39 @@
-export type Job = {
+import { InputLabelProps } from "@mui/material";
+
+type selectOptions = {
   name: string;
-  audioFile: File | string | undefined;
-  participants: number;
-  language: string;
-  status: string;
+  value: string | number;
 };
 
-export type BulkJob = {
-  name: string;
+export interface BulkJob {
+  name: string,
```

Comment:
Not sure about this one...

Reply:
As `BulkJob` can be extended I used `interface` here.